lib/src/common/model/checks.dart

```diff
@@ -127,6 +129,7 @@ class CheckRun {
       checkSuiteId: input['check_suite']['id'],
       detailsUrl: input['details_url'],
       startedAt: DateTime.parse(input['started_at']),
+      conclusion: CheckRunConclusion._fromValue(input['conclusion']),
```

Comment:
Please add a test

Reply:
Hi Casey, I didn't find any unit test for this check file, do you want me to add a unit test for this class?

Reply:
Yes, please add the unit test

Reply:
Sure, I will add one now.